packages/create-cloudflare/templates/next/c3.ts

```diff
@@ -27,10 +28,30 @@ const generate = async (ctx: C3Context) => {
 
 	await runFrameworkGenerator(ctx, [projectName]);
 
-	copyFile(
-		join(getTemplatePath(ctx), "wrangler.toml"),
-		join(ctx.project.path, "wrangler.toml")
+	const wranglerToml = readFile(join(getTemplatePath(ctx), "wrangler.toml"));
+
+	const newToml = new MagicString(wranglerToml);
+
+	// Note: here we add `# KV Example:` to the toml file for the KV example, we don't actually
+	//       include the comment in the template wrangler.toml file just so to keep it identical
+	//       and consistent with that of all the other frameworks
+	//       (instead of making it a special case which needs extra care)
+	newToml.replace(
```

Comment:
PS: honestly I don't see any benefit in using `MagicString` here to do the replacing as Regex replacement would have worked just as well... but I figured I'd use `MagicString` to be consistent with the codebase 🤔

Reply:
ah ok cool, I guess I'll go old school and use a regex then 🙂👍